src/components/ui/InputText/InputText.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useRef, useEffect } from 'react'
+import { Input as UIInput, Label as UILabel } from '@faststore/ui'
+import type { InputProps } from '@faststore/ui'
+import Button from 'src/components/ui/Button'
+import IconButton from 'src/components/ui/IconButton'
+import Icon from 'src/components/ui/Icon'
+
+import './input-text.scss'
```

Comment:
After some recent changes, we're now importing the component's CSS file inside the file `components.scss`.

Reply:
Oh! Thanks for the heads up! I forgot about it! I'll move it there.